src/pe/mod.rs

```diff
@@ -267,17 +276,195 @@ impl<'a> PE<'a> {
     }
 }
 
+fn pad(length: usize, alignment: Option<usize>) -> Option<Vec<u8>> {
```

Comment:
This should go in some util or trait and be reused https://github.com/m4b/goblin/pull/361/files#r1349133982

Reply:
done